.github/workflows/libabigail.yaml

```diff
@@ -0,0 +1,112 @@
+name: Run Libabigail
+on: [pull_request]
+
+jobs:
+  build-release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Previous Release
+      uses: actions/checkout@v3
+    - name: Retrieve from Container
+      run: |
+       mkdir -p /tmp/libs
+       docker run --rm \
+          --workdir=$(pwd) \
+          --volume=/tmp/libs:/tmp/libs \
+          fluxrm/flux-sched:focal \
```

Comment:
This is great!

I'm sure there is a good reason, but why use the flux-sched docker image instead of flux-core here? The flux-sched image only gets updated when changes to flux-sched are merged, so it could be out of date?

Reply:
I honestly didn't know there was a flux core one :laughing: I always use flux-sched! Even better, I think we should use the containers (TBA) built via GitHub actions and pushed to ghcr.io #4996